packages/expo-sensors/android/src/main/java/expo/modules/sensors/modules/DeviceMotionModule.kt

```diff
@@ -168,17 +168,16 @@ class DeviceMotionModule : Module(), SensorEventListener2 {
 
   private fun eventsToMap(): Bundle {
     val map = Bundle()
-    var interval = 0.0
```

Comment:
@lukmccall Note that I replaced this with the Expo-managed update interval as It wasn't clear to me why interval would be populated with the timestamp, but only in this case.